package.json

```diff
@@ -26,11 +26,12 @@
     "OpenID Connect"
   ],
   "scripts": {
-    "build": "npm run build-esm && npm run build-node && npm run build-browser && npm run build-browser-min && npm run build-types",
-    "build-esm": "esbuild src/index.ts --bundle --outfile=dist/oidc-client-ts.mjs --format=esm",
-    "build-node": "esbuild src/index.ts --bundle --outfile=dist/oidc-client-ts.cjs --platform=node",
-    "build-browser": "esbuild src/index.ts --bundle --outfile=dist/oidc-client-ts.js --platform=browser --global-name=oidc",
-    "build-browser-min": "npm run build-browser -- --minify --outfile=dist/oidc-client-ts.min.js",
+    "build": "node scripts/build.js && npm run build-types",
+    "build2": "npm run build-esm && npm run build-node && npm run build-browser && npm run build-browser-min && npm run build-types",
+    "build-esm2": "esbuild src/index.ts --bundle --outfile=dist/oidc-client-ts.mjs --format=esm",
+    "build-node2": "esbuild src/index.ts --bundle --outfile=dist/oidc-client-ts.cjs --platform=node",
+    "build-browser2": "esbuild src/index.ts --bundle --outfile=dist/oidc-client-ts.js --platform=browser --global-name=oidc",
+    "build-browser-min2": "npm run build-browser -- --minify --outfile=dist/oidc-client-ts.min.js",
```

Comment:
Can we remove these `build-*2` scripts? Otherwise this looks good to me

Reply:
ups, sure